src/script/common/c_content.cpp

```diff
@@ -526,6 +526,9 @@ ContentFeatures read_content_features(lua_State *L, int index)
 	// Amount of light the node emits
 	f.light_source = getintfield_default(L, index,
 			"light_source", f.light_source);
+	if (f.light_source > LIGHT_MAX) {
+		f.light_source = LIGHT_MAX;
```

Comment:
It is normally emitted in lua. It is just here if somebody overwrites minetest.register_node().